docs/pages/guides/expo-ui-swift-ui.mdx

```diff
@@ -0,0 +1,326 @@
+
+</Collapsible>
+
+<Collapsible summary="How is Expo UI different from libraries like `react-native-paper` or `react-native-elements`?">
+
+Expo UI is not "yet another" UI library and not an opinionated design kit. Instead, it's a primitives library: it exposes native SwiftUI and Jetpack Compose components directly to JavaScript, rather than re-implementing or simulating UI in JavaScript.
+
+</Collapsible>
+
+<Collapsible summary="Can I use `@expo/ui/swift-ui` on Android or Web?">
+
+The first milestone for Expo UI is achieving a 1-1 mapping from SwiftUI to Expo UI. Universal support will come in the next stage of the roadmap. Our priority is to establish strong SwiftUI support first, and then expand to Jetpack Compose on Android and DOM support on the Web.
+
+</Collapsible>
+
+<Collapsible summary="Can I use React Native components inside SwiftUI components?">
+
+Yes, you can place React Native components as JSX children of Expo UI components. Expo UI automatically creates a [`UIViewRepresentable`](https://developer.apple.com/documentation/swiftui/uiviewrepresentable) wrapper for you.
+However, keep in mind that the SwiftUI layout system works differently from UIKit and has some limitations. According to Apple's documentation:
+
+> **Warning** SwiftUI fully controls the layout of the UIKit view's [`center`](https://developer.apple.com/documentation/UIKit/UIView/center), [`bounds`](https://developer.apple.com/documentation/UIKit/UIView/bounds), [`frame`](https://developer.apple.com/documentation/UIKit/UIView/frame), and [`transform`](https://developer.apple.com/documentation/UIKit/UIView/transform) properties. Don't directly set these layout-related properties on the view managed by a [`UIViewRepresentable`](https://developer.apple.com/documentation/swiftui/uiviewrepresentable) instance from your own code because that conflicts with SwiftUI and results in undefined behavior.
+
+Also note that once you render React Native components, you're leaving the SwiftUI context. If you want to add Expo UI components again, you'll need to reintroduce a `<Host>` wrapper.
+
+We recommend keeping SwiftUI layouts self-contained. Interop is possible, but it works best when boundaries are clearly defined.
+
+</Collapsible>
+
+<Collapsible summary="I’m a SwiftUI developer. Why should I learn Expo UI?">
+
+Because React's promise of _"learn once, write anywhere"_ now extends to SwiftUI and Jetpack Compose. With Expo UI, you can apply your SwiftUI knowledge to build apps that run in the React Native ecosystem, extend to the Web through [DOM components](/guides/dom-components/), and even integrate [2D](https://shopify.github.io/react-native-skia/) and [3D](https://github.com/wcandillon/react-native-webgpu) rendering. The system is flexible enough that different parts of your app can use different approaches — giving you seamless integration at the component level.
```

Comment:
```suggestion
Because React's promise of _"learn once, write anywhere"_, it now extends to SwiftUI and Jetpack Compose. With Expo UI, you can apply your SwiftUI knowledge to build apps that run in the React Native ecosystem, extend to the Web through [DOM components](/guides/dom-components/), and even integrate [2D](https://shopify.github.io/react-native-skia/) and [3D](https://github.com/wcandillon/react-native-webgpu) rendering. The system is flexible enough that different parts of your app can use different approaches &mdash; giving you seamless integration at the component level.
```